doc/gov/chapters/chapter05.md

```diff
@@ -1,18 +1,40 @@
 [<< Back](../)
 
-# 5. Chapter 5
+# 5. Concensus-based Decision Making
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [5.1 Introduction](#5.1)
-* [5.2 Heading](#5.2)
-* [5.3 Heading](#5.3)
+* [5.2 Voting representatives](#5.2)
+* [5.3 Voting Process](#5.3)
 
 <a name="5.1"></a>
 ## 5.1 Introduction
 
+Consensus decision making is the model adopted by CNTT for all decisions taken within CNTT. Buy in from all community member organisations is essential for the success of CNTT. 
+
+In situations where consensus cannot be reached, a reasonable efforts shall be taken made to identify and address the core root causes of the disagreements. If, in spite of these efforts, an agreement cannot be reached, a the voting process, described below in Section 5.3, shall be undertaken to resolve the issue.
+
 <a name="5.2"></a>
-## 5.2 Heading
+## 5.2 Voting representatives
+
+Each CNTT member organisation shall nominate a single voting representative. A list of voting representatives shall be maintained within CNTT<sup>1</sup>.
+
+Being on the voting representative list **doesn't** mean the right to vote. The following criteria<sup>2</sup> shall be used to identify eligible voting representatives prior to a voting event. To be eligible, the voting representative’s organization must have had people from their organization(or directly associated with them<sup>2</sup>) collectively satisfy the conditions below:
```

Comment:
Minor typo: need space between word and parentheses -- "... organization (or directly ..."

Reply:
fixed